src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,42 @@
+.clock {
+  @include pos;
+
+  &__arrow-min {
+    @include pos;
+    width: 10px;
+    height: 150px;
+    border-radius: 5px;
+    background-color: #c25d5d;
+    transform-origin: bottom;
+    animation: round 3600s linear infinite;
+  }
+
+  &__arrow-sec {
+    position: absolute;
```

Comment:
Why every element is absolute? Let the container be relative and all inner elements absolute